linux/tools.Dockerfile

```diff
@@ -49,22 +49,29 @@ RUN npm install -q -g @pnp/cli-microsoft365
 
 # Install Bicep CLI
 RUN curl -Lo bicep https://github.com/Azure/bicep/releases/latest/download/bicep-linux-x64 \
-  && chmod +x ./bicep \
-  && mv ./bicep /usr/local/bin/bicep \
-  && bicep --help
+    && chmod +x ./bicep \
+    && mv ./bicep /usr/local/bin/bicep \
+    && bicep --help
 
 # Temp: fix ansible modules. Proper fix is to update base layer to use regular python for Ansible.
 RUN mkdir -p /usr/share/ansible/collections/ansible_collections/azure/azcollection/ \
     && wget -nv -q -O /usr/share/ansible/collections/ansible_collections/azure/azcollection/requirements-azure.txt https://raw.githubusercontent.com/ansible-collections/azure/dev/requirements-azure.txt \
     && /opt/ansible/bin/python -m pip install -r /usr/share/ansible/collections/ansible_collections/azure/azcollection/requirements-azure.txt
 
-# Copy and run script to Install powershell modules and setup Powershell machine profile
-COPY ./linux/powershell/PSCloudShellUtility/ /usr/local/share/powershell/Modules/PSCloudShellUtility/
-COPY ./linux/powershell/ powershell
-RUN /usr/bin/pwsh -File ./powershell/setupPowerShell.ps1 -image Top && rm -rf ./powershell
+# Powershell telemetry
+ENV POWERSHELL_DISTRIBUTION_CHANNEL=CloudShell \
+    # don't tell users to upgrade, they can't
+    POWERSHELL_UPDATECHECK=Off
 
-# install powershell warmup script
-COPY ./linux/powershell/Invoke-PreparePowerShell.ps1 linux/powershell/Invoke-PreparePowerShell.ps1
+# Copy and run script to install Powershell modules and setup Powershell machine profile
+COPY ./linux/powershell/ powershell
+RUN /usr/bin/pwsh -File ./powershell/setupPowerShell.ps1 -image Base && \
```

Comment:
hmm, that part is for the base.Dockerfile?

Reply:
Yes:

https://github.com/Azure/CloudShell/blob/f3e3630c9a8840340761e403bf7dfcee6489e223/linux/base.Dockerfile#L194-L196